config/environment.js

```diff
@@ -1,5 +1,9 @@
 'use strict';
 
 module.exports = function(/* environment, appConfig */) {
-  return { };
+  return {
+    'ember-cli-mirage': {
+      trackRequests: true
```

Comment:
out of curiosity, what does this do?

Reply:
This makes pretender keep a collection of the requests that were made. I guess this used to always be the case. The tests use this collection to make a bunch of [assertions](https://github.com/elwayman02/ember-data-github/blob/2f6c41dba843e0aad916b2ab7e70bdd540ac971c/tests/acceptance/current-github-user-test.js#L24).